### ISSUE TITLE
macOS Sierra install error `tmutil` "Cannot set path-based exclusion for relative path"

### ISSUE THREAD

**Issue by dougnukem** [travis-ci/gimme#79]
I'm running into an error trying to install `go 1.7.4` on macOS Sierra

Installed latest `gimme` 1/7/2016
```
curl -sL -o ~/bin/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
chmod +x ~/bin/gimme
```

Ran:

```
GIMME_DEBUG=1 gimme install 1.7.4
+ GIMME_VERSION=v1.1.0
+ GIMME_COPYRIGHT='Copyright (c) 2016 Dan Buch, Tianon Gravi, Travis CI GmbH'
+ GIMME_LICENSE_URL=https://raw.githubusercontent.com/travis-ci/gimme/v1.1.0/LICENSE
++ uname -s
++ tr '[:upper:]' '[:lower:]'
+ : darwin
++ uname -s
++ tr '[:upper:]' '[:lower:]'
+ : darwin
++ uname -m
+ : x86_64
++ uname -m
+ : x86_64
+ : /Users/doug/.gimme/envs
+ : /Users/doug/.gimme/versions
+ : /var/folders/5g/vl_z2k017ml372k4zx7hv2x00000gn/T//gimme
+ : https://github.com/golang/go.git
+ : auto
+ : osx10.8
+ : https://storage.googleapis.com/golang
+ [[ darwin == mingw* ]]
+ [[ 2 -gt 0 ]]
+ case "${1}" in
+ break
+ [[ -n install ]]
+ GIMME_GO_VERSION=install
+ [[ -n 1.7.4 ]]
+ GIMME_VERSION_PREFIX=1.7.4
+ case "${GIMME_ARCH}" in
+ GIMME_ARCH=amd64
+ case "${GIMME_HOSTARCH}" in
+ GIMME_HOSTARCH=amd64
+ [[ install == \s\t\a\b\l\e ]]
+ _assert_version_given install 1.7.4
+ [[ -z install ]]
+ '[' ']'
+ unset GOARCH
+ unset GOBIN
+ unset GOOS
+ unset GOPATH
+ unset GOROOT
+ unset CGO_ENABLED
+ unset CC_FOR_TARGET
+ mkdir -p 1.7.4 /Users/doug/.gimme/envs
+ _exclude_from_backups 1.7.4
+ case "${GIMME_HOSTOS}" in
+ tmutil addexclusion 1.7.4
1.7.4: Cannot set path-based exclusion for relative path.
```

Error:

Seems to be related to some Mac OS specific `tmutil` Time Machine backup exclusion.

```
1.7.4: Cannot set path-based exclusion for relative path.
```

**Comment by philpennock** [comment]
I wrote this an am currently on Sierra.

Your invocation is off.  You've asked for a version of `install` and told it to install under the prefix `1.7.4`.  The code assumes that `$GIMME_VERSION_PREFIX` is an absolute path.

So the quick fix for _you_ is to change the command-line invocation to `gimme 1.7.4` and separately from that, we should try to make `gimme` more robust against this mishap.

**Comment by philpennock** [comment]
PR #80 is passing tests, but I'm not closing _this_ issue #79 until something has been merged to master to resolve the lack of robustness, whether that's PR #80 or some other solution.

**Comment by philpennock** [comment]
PR #80 has been merged, so I'm considering this #79 resolved.
